layouts/partials/banner.html

```diff
@@ -1,6 +1,6 @@
 {{ $name := "cloud-engineering-summit" }}
-{{ $summary := "Come join Pulumi for the industry's first Cloud Engineering Summit! October 7&ndash;8." }}
-{{ $url := "https://cloudengineeringsummit.com/?utm_campaign=summit&utm_source=pulumi.com&utm_medium=top-banner" }}
+{{ $summary := "The Cloud Engineering Summit replay is available! Watch all your favorite talks, on your schedule." }}
```

Comment:
Nit: What do you think of this slight variant?

```suggestion
{{ $summary := "The Cloud Engineering Summit replay is available! Watch all your favorite talks, on demand." }}
```